core/src/components/datetime/utils/format.ts

```diff
@@ -20,7 +20,14 @@ export const getLocalizedTime = (locale: string, refParts: DatetimeParts, use24H
     minute: 'numeric',
     timeZone: 'UTC',
     hour12: !use24Hour,
-  }).format(new Date(convertDataToISO(refParts)));
+  }).format(
+    new Date(
+      convertDataToISO({
+        ...refParts,
+        tzOffset: undefined,
```

Comment:
When fixing FW-1831, would this issue also get resolved? The solution for that issue may be to remove any timezone processing from `convertDataToISO`. If this issue would get resolved, it might be worth adding a TODO to make sure we remove this additional line since it would be redundant.

Reply:
Yes, should be resolved by any changes to `convertDataToISO` that ignores the timezone offset. I'll update to include the TODO.